unix_commands.sh

```diff
@@ -0,0 +1,14 @@
+echo Shaked >> a-1
```

Comment:
instead of using >> for redirection, you can use > to create the file and write the content in a single step.